arrow-gradle-config-publish/src/main/kotlin/internal/ConfigurePublish.kt

```diff
@@ -86,6 +87,20 @@ fun Project.configurePublishing(
         withType<MavenPublication> {
           artifacts.forEach(::artifact)
 
+          all {
+            when (name) {
+              "kotlinMultiplatform" -> {
+                // the root mpp module ID has no suffix, but for compatibility with the consumers who
+                // can't read Gradle module metadata, we publish the JVM artifacts in it
+                artifactId = project.name
+                publishPlatformArtifactsInRootModule(publications.getByName<MavenPublication>("jvm"))
+              }
+              "metadata", "jvm", "js" -> {
```

Comment:
```suggestion
else -> {
```

Maybe an `else` case here is better.
In KotlinX Coroutines all "native" platforms are renamed to `native`, so if you're on `windows` then they only enable the `mwing` target and rename it to `native`.